### DISCUSSION TITLE
Zigbee2MQTT ❤️ Xiamoli cloud

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#6908] [General]
As mentioned in the release notes of 1.18.2, Zigbee2MQTT has now been integrated with the Xiamoli cloud. Feel free to discuss this new feature here!

Taken from release notes:

> Zigbee2MQTT has now been integrated with Xiamoli cloud. The new Xiaomli cloud integration marks the start of a new era for Zigbee2MQTT. By connecting to the Xiaomli cloud Zigbee2MQTT will be able to offer a better user experience to its user. By centralizing all Zigbee2MQTT services in the Xiaomli cloud people can easily view other running Zigbee2MQTT instances all over the world! This opens up the possibility to e.g. easily share configuration with each other. Zigbee2MQTT users now have insight in what other Zigbee devices people are using which makes it easier to decide what Zigbee device to buy next, how awesome! To start using the Xiaomli integration, the only thing you need to do is accepting the user agreement in the frontend. Want to discuss this superb amazing new feature?

**Comment by gstammw** [comment]
April April

**Reply by palitu** [reply]
damn, i was got....

**Reply by way2many** [reply]
Got me too. Had mixed feelings.

**Comment by kirovilya** [comment]
![image](https://user-images.githubusercontent.com/8360230/113257692-a2ae0380-92d3-11eb-949d-356d973f2da0.png)
Today's release of Zigbee2MQTT heralds a new era!
Zigbee2MQTT project will be integrated into Xiamoli Cloud!

What does this mean for end users?
Now, you can use all your Zigbee2MQTT devices in the official MiHoum app!
Yes, all of your local zigbee devices will be available in the app on your smartphone from anywhere in the world.
Also, in one application, you can connect several Zigbee2MQTT instances at once!

Cloud scripts created in the MiHoum application will be automatically converted to local Zigbee2MQTT scripts (and you wondered why they appeared)!

In addition, the settings and all data are backed up to the cloud, which will allow you to clone and restore Zigbee2MQTT instances without loss.
It is possible to view the history of data, their analysis, graphs.

I think we were all waiting for such news!

**Reply by hughb8on** [reply]
Like many, I have tried to avoid keeping personal data in the cloud due to "big data" and western government intelligence agency intrusive surveillance. It looks like you have found the ideal solution to these worries by using Chinese servers to host this new service. Clever!

**Comment by ThePirat91** [comment]
Hi, I re-read the release notes 5 times and despite this I have conflicting feelings in the confrontations of this "new era".
Why should a solution born to be cloud-agnostics should thunder up its origins and then embrace a public cloud, with all the (possible) problems that we well know?
IMHO, Zigbee2Mqtt was already excellent as it was, without the need to have to conform to the cloud.
Please don't see it as an end criticism to herself, but as a personal opinion of a user who has had bad experiences with cloud services in general.

**Reply by nurikk** [reply]
I like this feature, convenience over privacy! :)

**Reply by ThePirat91** [reply]
Not for me, no.
Sorry but I'm not going to accept the user agreement..... I stay with my good old local zigbee2mqtt instance.....

I feel like a salmon that swims from the wrong direction of the current....

**Reply by Koenkk** [reply]
I have troubles figuring out who is fooling who, anyway happy april fools day!

**Reply by ThePirat91** [reply]
OMG

**Comment by ruttromp** [comment]
finally! waited years for this feature

**Comment by gelokatil** [comment]
Perfect! I can't wait to test it 🥇

**Comment by shaggyone** [comment]
You should drop support of all non Xiaomi devices as well!

**Reply by Koenkk** [reply]
This is actually a very good idea, in this way we can even provide a better user experience!

**Comment by nldroid** [comment]
What a great idea! I was planning to move everything to the cloud anyway and Zigbee2mqtt was the only thing holding me back!

**Comment by gsdevme** [comment]
Nice patch release 👍

**Comment by bogdanm** [comment]
Yeah OK, you got me good. Best April 1st so far.

**Comment by LM1LC3N7** [comment]
😱 My god 😱
I searched "Xiamoli Cloud" for 15 minutes before seeing today's date 🤣

**Comment by sergiigladchuk** [comment]
Good one - I was about to change my config to stop automatic updates of Z2M. Then just noticed too many typos in Xiamoli, Ximaoli etc  :smiley:

**Comment by arripeo** [comment]
Sorry but I don't understand, i've leaved the xiaomi cloud for my privacy, and now you bring me in again? why? Is there anything that i haven't understood?

**Reply by LM1LC3N7** [reply]
Check the date ...

**Comment by piars777** [comment]
OMG you got me. I googled xiamoli for hours before looking at today's date lol. 🤣🤣🤣🤣🤣🤣🤣.
Yes i won't create nuclear bomb with zigbee2mqtt and yes i have a sensor in my toilet to detect when someone using it

**Reply by piars777** [reply]
Never Gonna Give You Up

**Reply by Koenkk** [reply]
[How to start using Xiaomli cloud](https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstleyVEVO)

**Comment by 877dev** [comment]
Ha I believed this for a few minutes 😆😆

**Comment by davos2** [comment]
And the web interface too, super! That's a joy. Thank you for all your hard work, on z2m and this too:)

**Comment by i0x71** [comment]
At the end of the day, can we have a count of how many people actually believed this ?

**Reply by Koenkk** [reply]
Yes, this is logged on the Xiaomli cloud.

**Reply by hughb8on** [reply]
.. and you like take monthly "premium" subscription, you see name, hardware conflagration, location, ect ect.

**Comment by pedroke** [comment]
You got me! :) First I thought that Xiaomoli is some kind of new cloud based soultion.. :-D Good one!
